tools/pipelines/templates/upload-json-steps.yml

```diff
@@ -29,42 +29,10 @@ steps:
   clean: true
 
 # Download the api-extractor outputs
-- template: download-api-extractor-artifact.yml
```

Comment:
It occurs to me: is this step actually necessary? This template downloads a bunch of API model bundles, merges them, and uploads them under another artifact name. But now we're only downloading 1 bundle.

How easy would it be to update the website deployment logic to just pull down the appropriate artifacts published by the `build - client` pipeline?

Reply:
Not for this PR (unless it's a trivial change), just seems like this step is no longer strictly necessary.

Reply:
Yeah I think this could be an improvement. The only part that might be a bit tricky implementation-wise is targetting the release branch build artifact rather than the running branch artifact. Ideally, we wouldn't need to explicitly input the exact release branch name in order to target it's build artifact.